sansio_lsp_client/client.py

```diff
@@ -118,42 +119,39 @@ class ClientState(enum.Enum):
             'dynamicRegistration': True,
         },
         'references': {
-            'dynamicRegistration': True
+            'dynamicRegistration': True,
+        },
+        '!!! SEE': '',
```

Comment:
Somehow this seems different from everything else. Is that intentional?

Reply:
Not intentional, will remove. 

(I'm just learning git branches and failing :) Will be trying to add tests)

Reply:
I recently wrote about branches to my git-guide, if it helps: https://akuli.github.io/git-guide/branches.html

Feel free to make issues to git-guide or ask help here.

Reply:
Thanks, will give it a read